tasks/slurmdbd.yml

```diff
@@ -4,6 +4,7 @@
   ansible.builtin.package:
     name: "{{ __slurm_packages.slurmdbd }}"
     state: "{{ 'latest' if slurm_upgrade else 'present' }}"
+  when: slurm_apt_repository is not defined
```

Comment:
Not sure to understand.
This won't install slurmdbd If `slurm_apt_repository` is defined. Why?

Reply:
it is in the same logic not to have to install slurm packages twice when it is compiled

Reply:
fixed